scripts/main/lychee.js

```diff
@@ -252,7 +252,7 @@ lychee.init = function () {
 			lychee.admin = !lychee.api_V2;
 			lychee.nsfw_visible_saved = lychee.nsfw_visible;
 
-			lychee.upload_processing_limit = parseInt(data.config.upload_processing_limit);
+			lychee.upload_processing_limit = parseInt(data.config.upload_processing_limit) || 4;
```

Comment:
This is wrong. This prevents the value of `0` from being used. I don't think we can use this `||` syntax for numerals that can be `0`.

Reply:
Hummmm indeed, I did not think about this. The problem is that if you send `null` to `parseInt` you get `NaN` which obviously crash all the upload checks afterwards.